quickstart-services.yml

```diff
@@ -86,7 +86,7 @@ services:
       - 4456
     environment:
       - LOG_LEVEL=debug
-    restart: on-failure
+    restart: unless-stopped
```

Comment:
@valentinyanakiev can you verify this one?

Reply:
I'll move that in separate PR.